cli/command_snapshot_create.go

```diff
@@ -129,6 +133,32 @@ func snapshotSingleSource(ctx context.Context, rep *repo.Repository, u *snapshot
 
 	manifest.Description = *snapshotCreateDescription
 
+	if *snapshotCreateStartTime != "" {
+		startTime, err := time.Parse(timeFormat, *snapshotCreateStartTime)
+
+		if err != nil {
+			return errors.Wrap(err, fmt.Sprintf("Could not parse start-time, should be like the format '%s'", timeFormat))
+		}
+
+		manifest.StartTime = startTime
+
+		if *snapshotCreateEndTime == "" {
+			manifest.EndTime = startTime
+		}
+	}
+
+	if *snapshotCreateEndTime != "" {
+		endTime, err := time.Parse(timeFormat, *snapshotCreateEndTime)
+
+		if err != nil {
+			return errors.Wrap(err, fmt.Sprintf("Could not parse end-time, should be like the format '%s'", timeFormat))
+		}
+
+		if *snapshotCreateStartTime == "" {
+			manifest.StartTime = endTime
```

Comment:
i'm thinking: if somebody specifies one flag but not both, we can take actual snapshot duration and compute the other. would that be useful?

Also can you add one or two test cases for this logic? we already have CLI integration tests for a whole bunch of cases.

Reply:
It wouldn't be useful for my use case, but I'm happy to work out how to do it :)

I'll add tests as well, the existing ones serve as a good example

Reply:
try `tests/end_to_end_test/snapshot_create_test.go`